src/libs/actions/Report.ts

```diff
@@ -6071,6 +6071,20 @@ function resolveConciergeCategoryOptions(
     } as Partial<ReportActions>);
 }
 
+function setOptimisticTransactionThread(reportID?: string, parentReportID?: string, parentReportActionID?: string, policyID?: string) {
```

Comment:
Can you add a plain English explanation for what this is used for? I think it can be confused with the create optimistic transaction thread which is not what we do here